src/CMakeLists.txt

```diff
@@ -0,0 +1,323 @@
+cmake_minimum_required(VERSION 3.2)
+project(QZXing)
+
+find_package(Qt5 REQUIRED Core)
+find_package(Qt5 REQUIRED Network)
+find_package(Qt5 REQUIRED Xml)
+find_package(Qt5 REQUIRED Multimedia)
+find_package(Qt5 REQUIRED MultimediaWidgets Widgets)
+find_package(Qt5 REQUIRED Svg Quick QuickControls2)
```

Comment:
Usage of Quick and Multimedia should be configurable, as it is currently with `CONFIG += qzxing_qml` and `CONFIG += qzxing_multimedia`